### ISSUE TITLE
8.5, Table 9 (PDF Only): Missing part of Row 3

### ISSUE THREAD

**Issue by ERyan71258** [SMPTE/st430-5#112]
The HTML displays Table 9 as follows:
![image](https://github.com/SMPTE/st430-5/assets/118845840/2e201709-a867-49dd-aef8-60ffbbc69d60)

The PDF displays Table 9 as follows. The first line of Row 3 (AssetHashError) is missing.
![image](https://github.com/SMPTE/st430-5/assets/118845840/3d50d03f-0826-473f-b0eb-0b5beed373f5)

**Comment by SteveLLamb** [comment]
@ERyan71258 this appears to be resolved, however I see the cell breaking across the pages now. I think that should not happen? If you agree, I'll create an issue on the tooling for that.

**Comment by ERyan71258** [comment]
Yes, cells (i.e., rows) should not break across pages. There have been occasional exceptions where  tables contain long blocks of code that extend over more than one page. So there should be the ability to turn off the no-cell-break rule.

And if possible, to control:
- Where to break a table at the row level, i.e., (send a row to the next page).
- In the instance where a row itself must extend to the next page, then control where that break within the row occurs.

**Comment by SteveLLamb** [comment]
I have noted the above in both above issues within the tooling. Leaving this open as duplicate on this document until resolved.
